src/test/client/TestClient.h

```diff
@@ -20,8 +20,52 @@
 #include "msg/Messenger.h"
 #include "mon/MonClient.h"
 #include "osdc/ObjectCacher.h"
-
+#include "client/MetaRequest.h"
 #include "client/Client.h"
+#include "messages/MClientReclaim.h"
+#include "messages/MClientSession.h"
+
+#define dout_subsys ceph_subsys_client
+
+class ClientScaffold : public Client {  
+public:
+    ClientScaffold(Messenger *m, MonClient *mc, Objecter *objecter_) : Client(m, mc, objecter_) {}
+    ~ClientScaffold();
+    int check_dummy_op(const UserPerm& perms){
+      RWRef_t mref_reader(mount_state, CLIENT_MOUNTING);
+      if (!mref_reader.is_state_satisfied()) {
+        return -CEPHFS_ENOTCONN;
+      }
+      MetaRequest *req = new MetaRequest(CEPH_MDS_OP_DUMMY);
+      int res = make_request(req, perms);
+      ldout(cct, 10) << __func__ << " result=" << res << dendl;
+      return res;
+    }
+    int check_client_eviction() {
+      RWRef_t mref_reader(mount_state, CLIENT_MOUNTING);
+      if (!mref_reader.is_state_satisfied()) {
+        return -CEPHFS_ENOTCONN;
+      }
+      auto session = _get_or_open_mds_session(0);
+      auto msg = make_message<MClientSession>(CEPH_MDS_OP_EVICT_CLIENT, session->seq);
+      int res = session->con->send_message2(std::move(msg));
+      ldout(cct, 10) << __func__ << " result=" << res << dendl;
+      return res;
+    }
+    int check_unknown_reclaim_flag() {
```

Comment:
```suggestion
    int check_unknown_reclaim_flag(int flag) {
```

so you can test both `-1` and `0`.

Reply:
great idea, pushed the change. PTAL